includes/admin/views/html-notice-maxmind-license-key.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Admin View: Notice - Missing license key
+ *
+ * @package WooCommerce\Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+?>
+
+<div id="message" class="updated woocommerce-message">
+	<a class="woocommerce-message-close notice-dismiss" href="<?php echo esc_url( wp_nonce_url( add_query_arg( 'wc-hide-notice', 'maxmind_license_key' ), 'woocommerce_hide_notices_nonce', '_wc_notice_nonce' ) ); ?>"><?php esc_html_e( 'Dismiss', 'woocommerce' ); ?></a>
+
+	<p>
+		<strong><?php esc_html_e( 'Geolocation has not been configured.', 'woocommerce' ); ?></strong>
+	</p>
+
+	<p>
+		<?php
+		echo wp_kses_post(
+			sprintf(
+				/* translators: %s: integration page */
+				__( 'You must enter a valid license key on the <a href="%s">MaxMind integration settings page</a> in order to use the geolocation service.', 'woocommerce' ),
```

Comment:
This was something @claudiosanches raised as well. The geolocation feature as a whole isn't very well explained, and a lot of people seem to use it when they don't really need to. I'm not sure the best way to go about explaining this to the user though. If we're moving the integration away from core, then the notice would be fine (since it's opt-in to the plugin), but as-is, I can agree we could add clarity to that.

Reply:
Notably, different states in the USA have different tax rates though, so I'm not sure how best to explain the use cases where geolocation is a good fit.

Reply:
Maybe we add something like "Note: if you only sell to one country, you might not need MaxMind geolocation service and, instead of getting a license key, you can disable geolocation in the general settings tab. For more information, see the documentation (link to a page on docs.woocommercecom explaining all the things related to MaxMind geolocation)."

What do you think?

My understanding is that the geolocation provided by WooCommerce core works only to determine the country, so it is not enough for USA stores that need to calculate different taxes depending on the state. But maybe I'm missing something?

Reply:
@ObliviousHarmony ignore my previous comment. After writing it, I saw your commit where you added some clarity to the message. Looks good to me now. Thanks!